domain/consensus/processes/dagtraversalmanager/dagtraversalmanager_test.go

```diff
@@ -3,11 +3,113 @@ package dagtraversalmanager_test
 import (
 	"github.com/kaspanet/kaspad/domain/consensus"
 	"github.com/kaspanet/kaspad/domain/consensus/model/externalapi"
+	"github.com/kaspanet/kaspad/domain/consensus/model/testapi"
 	"github.com/kaspanet/kaspad/domain/consensus/utils/testutils"
 	"github.com/kaspanet/kaspad/domain/dagconfig"
 	"testing"
 )
 
+const sizeOfTheChainThatCommonToAllGraphs int = 5
```

Comment:
This could be shortened to something like "commonChainSize".
Also: you don't have to specify int (it's implied, and the go compiler can understand it by it's own by looking at the value)